### ISSUE TITLE
Request: Enhancements to access Log Viewer

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#3510]
At some point, please can we have a few enhancements for the log viewer?

According to the manual
'To view log files, navigate to the /Logs folder on eMMC or the sd card with File Explorer, then tap on the desired log file and select open.'

The list of files is then presented in alphabetical order (which normally makes sense), however, at the field, typically someone is going to want to look at the most recent log file for a flight they've just had.

1. Would it be possible to have an easy way to navigate to the current log file - perhaps from an SF, or even better to simply open the current log file from an SF.
2. It would be useful to have the option to list the log files in chronological order with most recent at the top. That would make them much more useful. When out flying when you'll typically want to see a very recent log. (I recognise the name format used is chrono within model name, but pure chrono would be more useful at the field when we swap between models.)
3. The file names could be made easier to understand and use, currently they are of format modelname-yyyy-mm-dd-hh-mm-ss-ms.csv which could be easier to interpret, I'm guessing spaces aren't allowed in file name so perhaps yyyy-mm-dd_hh:mm:ss:ms or if colon isn't allowed either then yyyy-mm-dd_hh-mm-ss-ms so it is a bit clearer where the date ends and the time starts.
4. With a long model name the file name becomes unhelpful as it is truncated both on the left side of the file manager screen and the right side so multiple filenames may appear to have the same (truncated) name. There appears to be plenty of room on the right side of the screen to move the Filename: up a few lines so the filename can be presented in its entirety.
5. Once the Filename: on the right has been moved up we then have room for a time stamp to be added. I'd suggest have 'Last modification: on one line and the next line to simply have yyyy-mm-dd  hh:mm:ss   (I don't think we need ms here) the difference in format i.e. - for date and : for time would make it easier to understand and I think is more conventional.

Many thanks!

**Comment by lthole** [comment]
Excellent suggestions!

**Comment by koakumaping** [comment]
Excellent suggestions!

**Comment by koakumaping** [comment]
![88135969d1f20c56abd5a374bbf7239](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/4099388/f6b41a17-be73-4a25-883b-965849c6dbd2)

**Comment by RealTadango** [comment]
My ideas:
 - Make a log folder per model
 - Order files by Date Descending so the newest one is always on top
 - Use sticks in the log viewer for zooming / scrolling and cursor control just like LView has
 - Add min / max stats and indicators

**Comment by LeoCheung123** [comment]
Sort log files by date created with most recent on top PLEASE. 

I fly three times a week, dozens of models in a year...now i have to turn the wheel a few hundred times to get to the log of the last flight🤣

**Comment by RealTadango** [comment]
Still needed

**Comment by lthole** [comment]
Still needed

**Comment by LeoCheung123** [comment]
Yes please. Half of the reason telemetry is needed is so we can review numbers carefully.

**Comment by bsongis-frsky** [comment]
<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/0a510d52-2cff-4be1-92e7-6cb5abb3e205" />

**Comment by bsongis-frsky** [comment]
Ready in the next nightlies
